pkg/util/stmtsummary/v2/record.go

```diff
@@ -119,6 +119,10 @@ type StmtRecord struct {
 	SumBackoffTimes      int64               `json:"sum_backoff_times"`
 	BackoffTypes         map[string]int      `json:"backoff_types"`
 	AuthUsers            map[string]struct{} `json:"auth_users"`
+	// RU
+	WRU            float64       `json:"wru"`
```

Comment:
those feilds are never been used?

Reply:
Fixed. Removed the partial work for adding ru in `information_schema.statements_summary`, it's should be in a separate PR.